examples/README.md

```diff
@@ -0,0 +1,23 @@
+# Examples
+
+This folder contains different examples of Strimzi custom resources and demonstrates how they can be used.
+
+* [Kafka cluster examples](./kafka)
```

Comment:
do we need to have "examples" close to each topic? we know to be in the examples folder I guess.

Reply:
Fixed.